### ISSUE TITLE
[Suggestion] Please make a video guide

### ISSUE THREAD

**Issue by dares9** [seanlowe/obsidian-timelines#88]
Hey! First of all, I want to really thank you for making this plugin and revamping the old concept. Thank you for all hard work.

My issue isn't really any critique of functionality, but my own; dealing with codes and tags, I'm following your guidelines for few hours now and can't for the life of me crack it out - I haven't been able to build a single timeline.

I know the issue is on me, but still, what seems like a great useful tool, is behind a wall for me, because it's just too hard to understand how to do it. 

If you'd made a video showcasing exactly what and where to click, it'd be a tremendous help. It doesn't have to be a super-polished video, just 2 minutes of instructions. I'm sure many people would appreciate! 

Thank you!

**Comment by seanlowe** [comment]
No worries. This is a complicated plugin to be sure. I'll try to get a video created soon-ish

**Comment by dares9** [comment]
Whoa, thank you for this sm! Will definitely try it out now. <3

**Comment by gleyconJ** [comment]
Friend this complement is exceedingly complicated, but I managed to use it with humility. I decided to use only frontmatter. And for me nothing that does not pose to be done for this language I simply ignore my life. This way I am using the feature.

 Even if I don't have 100% of the power he can give me, I can still do a lot. I tell you that more than 50% of what he could make me.

 Therefore. Everything is much simpler.
